pkg/store/proxy.go

```diff
@@ -289,8 +294,9 @@ func (s *ProxyStore) Series(originalRequest *storepb.SeriesRequest, srv storepb.
 
 	for _, st := range stores {
 		st := st
-
-		storeDebugMsgs = append(storeDebugMsgs, fmt.Sprintf("store %s queried", st))
+		if s.debug {
+			storeDebugMsgs = append(storeDebugMsgs, fmt.Sprintf("store %s queried", st))
```

Comment:
Should we add the debug log under `if s.debug` as well?

Reply:
You mean the one at L319 where these messages are printed? I assumed the leveled debugger would take care of that without much overhead.

Or do you want to add a debug log message here to indicate that we are appending data to `storeDebugMsgs` ?